cms/envs/common.py

```diff
@@ -405,6 +405,19 @@
     # .. toggle_tickets: https://openedx.atlassian.net/browse/DEPR-58
     # .. toggle_status: supported
     'DEPRECATE_OLD_COURSE_KEYS_IN_STUDIO': True,
+
+    # .. toggle_name: ENABLE_LIBRARY_AUTHORING_MICROFRONTEND
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True to enable the Library Authoring MFE will take the user to the new library authoring MFE.
```

Comment:
Could you reword the toggle description?

Reply:
Yeah, looks like a copy-paste malfunction.  Fixed.